### ISSUE TITLE
the example code has a bug "service_account" it should be "serviceaccount"

### ISSUE THREAD

**Issue by Jaafar-Nasrallah** [pulumi/pulumi-hugo#3145]
File: [themes/default/content/registry/packages/gcp/api-docs/serviceAccount/account/_index.md](https://www.pulumi.com/registry/packages/gcp/api-docs/serviceaccount/account/)

**Comment by scottslowe** [comment]
@Jaafar-Nasrallah Thanks for opening this issue! Can you clarify which language (TypeScript, Python, Go, C#, Java, or YAML) has the bug? I'm guessing Python, but if you could clarify that would be helpful. Thanks!

**Comment by Jaafar-Nasrallah** [comment]
Yes, as you have guessed its python. I would be glad to try and fix such issues and open a PR. But would such a small task be easier as only an issue?

**Comment by scottslowe** [comment]
@Jaafar-Nasrallah Thanks for confirming the language. If I'm not mistaken, this content is auto-generated so the fix may be more involved than it might seem. For now, opening this issue is sufficient. I'll move this to the repo for the GCP provider so that team can look at the docs auto-generation code for a fix. Thank you!
